docs/website/docs/guides/deployment-configurations/cpu.md

```diff
@@ -149,13 +149,13 @@ iree-compile \
 
 !!! tip "Tip - CPU targets"
 
-    The `--iree-llvmcpu-target-triple` flag tells the compiler to generate code
+    The `--iree-llvm-target-triple` flag tells the compiler to generate code
```

Comment:
> My iree-compiler version is 20230209.425

Why are you on such an old version? Maybe using Python 3.9/3.10 on macOS/Windows? (We have newer versions available on Python 3.11)

The latest flag names are indeed `--iree-llvmcpu-*`. They were changed in https://github.com/openxla/iree/pull/12564

Reply:
Thank you! Maybe my python is 3.9 and just run `python -m pip install iree-compiler`. I don't see the python requrements in the documents.

Reply:
Hi @ScottTodd Do we need to specify in the document that currently **only Python version 3.11** is supported.

Reply:
Probably worth adding that as a stop-gap. A few people have had the same issue (installing ooooold versions).

@stellaraccident what would you suggest? Could we start publishing Python 3.9/3.10 packages on Windows and macOS again? Or maybe mark the existing published packages as deprecated on those versions?